src/test/java/com/fastcampus/projectboard/domain/test/ArticleAuditingFieldsTest.java

```diff
@@ -0,0 +1,98 @@
+package com.fastcampus.projectboard.domain.test;
+
+import com.fastcampus.projectboard.domain.TestArticle;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("[AuditingField] AuditingFields equals hashcode 메서드 테스트")
+public class ArticleAuditingFieldsTest {
+
+    private LocalDateTime now;
+
+    @BeforeEach
+    void timeInit() {
+        now = LocalDateTime.now();
+    }
+
+    @DisplayName("서로다른 5개의 객체 중복 비교")
+    @Test
+    void givenTestArticles_whenDistinct_thenNotDuplication() {
+        // given
+        List<TestArticle> testArticles = getDiffFiveArticles();
+
+        // when
+        List<TestArticle> removeDuplicateData = testArticles.stream().distinct().toList();
+
+        // then
+        assertThat(removeDuplicateData.size()).isEqualTo(testArticles.size());
+    }
+
+    @DisplayName("모든 필드값이 같은 2개의 객체 중복 비교")
+    @Test
+    void givenTwoSameTestArticle_when_thenSameTestArticle() {
+        // given
+        TestArticle testArticle1 = getTestArticleDiff(1);
+        TestArticle testArticle2 = getTestArticleDiff(1);
+
+        // when
+
+        // then
+        assertThat(testArticle1).isEqualTo(testArticle2);
+        assertThat(testArticle1.hashCode()).isEqualTo(testArticle2.hashCode());
```

Comment:
실행문과 검증문이 일치한다면 주석으로 `// when & then` 스타일을 쓰는 것도 좋습니다.

```suggestion
        // when & then
        assertThat(testArticle1).isEqualTo(testArticle2);
        assertThat(testArticle1.hashCode()).isEqualTo(testArticle2.hashCode());
```

Reply:
넵 감사합니다. 지난번에 확인했었는데 정확히 사용법을 몰라서 사용하지 못했습니다.
Commit suggestion 하겠습니다.